Dockerfile

```diff
@@ -19,10 +20,13 @@ RUN apt-get update && apt-get install -y \
       python-requests \
       software-properties-common
 
+RUN apt-add-repository -y ppa:kevinkreiser/prime-server
```

Comment:
you dont need this i added `prime-server` to the `valhalla-core/valhalla` ppa for convenience